packages/ag-charts-website/src/content/docs/error-bars/index.mdoc

```diff
@@ -54,7 +57,7 @@ data: [
         pressure: 1.2,
         pressureLower: 1.1,
         pressureUpper: 1.4
-    },
+    }, ...
```

Comment:
Intentional?

Reply:
Yes. The intent here is to say that the `data` array includes 1 or more entries that follow this pattern.

Open to feedback on how to better convey this.